cmd/miniooni/main.go

```diff
@@ -224,13 +224,15 @@ func main() {
 	var inputs []string
 
 	if name == "web_connectivity" {
+		log.Info("Start fetch test lists")
 		list, err := testlists.NewClient(sess).Do(ctx, sess.ProbeCC())
 		if err != nil {
 			log.WithError(err).Fatal("cannot fetch test lists")
 		}
 		for _, entry := range list {
 			inputs = append(inputs, entry.URL)
 		}
+		log.Infof("{ %v } URLs received",len(inputs))
```

Comment:
This is not necessary. More details can be deduced using `-v` or from subsequent progress output.